content/kubernetes/re-clusters/upgrade-redis-cluster.md

```diff
@@ -78,62 +83,96 @@ customresourcedefinition.apiextensions.k8s.io/redisenterprisedatabases.app.redis
 deployment.apps/redis-enterprise-operator configured
 ```
 
-### Verify that the operator is running
+### Reapply other manual configurations
+
+When upgrading your cluster, you need to manually reapply the `ValidatingWebhookConfiguration` for the admission controller. See the [Enable the admission controller]({{<relref "/kubernetes/deployment/quick-start#enable-the-admission-controller">}}) step during deployment for more details.
```

Comment:
"_When upgrading your cluster_" - I think it's more accurate to say "_When upgrading your operator_" (or, "_the operator_"?). In practice, this isn't likely too make any difference for the reader, but since we do make the distinction between upgrading the operator and upgrading the cluster, and the installation of the webhook/SCCs are more of a part of installing the operator - I'd say we should refer the operator.